worker/test/src/RTC/Codecs/TestVP8.cpp

```diff
@@ -5,6 +5,8 @@
 
 using namespace RTC;
 
+constexpr uint16_t kMaxPictureId = (1 << 15) - 1;
```

Comment:
why the `k` prefix? :-). We don't use it in the codebase.

Reply:
Please replace with `MaxPictureId`.

Reply:
My fault. I did a quick search and found some of them. But they were inside the libwebrtc folder.